src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -4,14 +4,20 @@
  * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
  */
 
-#include <toolkits/drawing_classifier/drawing_classifier.hpp>
 
 #include <iostream>
+#include <random>
+#include <memory>
+#include <sstream>
 
+#include <ml/neural_net/compute_context.hpp>
+#include <ml/neural_net/model_backend.hpp>
+#include <ml/neural_net/model_spec.hpp>
 #include <core/logging/assertions.hpp>
 #include <core/logging/logger.hpp>
 #include <model_server/lib/variant_deep_serialize.hpp>
 #include <toolkits/evaluation/metrics.hpp>
+#include <toolkits/drawing_classifier/drawing_classifier.hpp>
```

Comment:
This line should be first. It was correct before.

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
oh, good to know! I thought I followed google style before.